tests/test_wave_05.py

```diff
@@ -70,31 +67,49 @@ def test_create_goal(client):
             "title": "My New Goal"
         }
     }
-
-@pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    pass
     # Act
-    # ---- Complete Act Here ----
+    response = client.put("/goals/1", json={
+        "title": "Update Goal Title"
+    })
+    response_body = response.get_json()
 
     # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
-    # ---- Complete Assertions Here ----
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body == {
+    "goal": {
+        "id": 1,
+        "title": "Update Goal Title"
+        }
+    }
+    goal = Goal.query.get(1)
+    assert goal.title == "Update Goal Title"
+
+
+# def test_update_task_not_found(client):
+#     # Act
+#     response = client.put("/tasks/1", json={
+#         "title": "Updated Task Title",
+#         "description": "Updated Test Description",
+#     })
+#     response_body = response.get_json()
+
+#     # Assert
+#     assert response.status_code == 404
+#     assert response_body == None
```

Comment:
```suggestion
```

I think you copied this over to use in writing the goal test, but it's not needed now.